refine readme (#371)
Oneflow-Inc/diffusers: README.md

[![PyPI version](https://badge.fury.io/py/onediff.svg)](https://badge.fury.io/py/onediff)
[![Docker image build](https://github.com/Oneflow-Inc/diffusers/actions/workflows/sd.yml/badge.svg)](https://github.com/Oneflow-Inc/diffusers/actions/workflows/sd.yml)

> # OneDiff (OneFlow Diffusers)

Reviewer: ```suggestion
# OneDiff
```

> A drop-in acceleration lib for diffusers. (Or other Stable Diffusion Models)

Reviewer: ```suggestion
A drop-in acceleration lib for diffusers and other diffusion models
```

> **_NOTE:_** OneDiff Enterprise version delivers even higher performance and second-to-none deployment flexibility.

## Features
- Acceleration for SDXL, SD2.1 and so on
  - [SDXL](https://github.com/Oneflow-Inc/diffusers/blob/a38c5ea475c07b4527981ec5723ccac083ed0a9c/examples/text_to_image_sdxl.py#L16) 
  - [SD2.1](https://github.com/Oneflow-Inc/diffusers/blob/a38c5ea475c07b4527981ec5723ccac083ed0a9c/examples/text_to_image.py#L30C10-L30C10)
- Convert PyTorch models like UNet to an OneFlow graph with one function
  - [oneflow_compile](https://github.com/Oneflow-Inc/diffusers/blob/a38c5ea475c07b4527981ec5723ccac083ed0a9c/examples/text_to_image_sdxl.py#L53)
- Enable multi-resolution input with the same compiled model
  - [The Multi graph way](https://github.com/Oneflow-Inc/diffusers/blob/a38c5ea475c07b4527981ec5723ccac083ed0a9c/examples/text_to_image_sdxl_save_load.py#L65)
  - [The Single graph way](https://github.com/Oneflow-Inc/diffusers#business-inquiry)
- Save/load/change device of the compiled graph
  - [Save and Load](https://github.com/Oneflow-Inc/diffusers/blob/a38c5ea475c07b4527981ec5723ccac083ed0a9c/examples/text_to_image_sdxl_save_load.py#L76)

>   - [Change device to do multi-process serving](https://github.com/Oneflow-Inc/diffusers/blob/a38c5ea475c07b4527981ec5723ccac083ed0a9c/examples/text_to_image_sdxl_mp_load.py#L52)

Reviewer: features每一项都有几个链接感觉有点太多看不过来，还有这些URL最好改成main分支的？
